kubetest/README.md

```diff
@@ -177,3 +180,17 @@ skew directory, potentially to upgrade/downgrade kubernetes to another version.
 [kubekins-e2e]: /jenkins/e2e-image
 [kubekins-e2e-prow]: /images/e2e-prow
 [prow]: /prow
+
+## Example Commands
+
+A simple example is as follow:<br>
+`kubetest --test --test_args=--ginkgo.focus=\[Feature:MasterUpgrade\] --upgrade-target=ci/latest-1.6 --report-dir=/workspace/_artifacts --disable-log-dump=true --report-prefix=upgrade --v=true --check-version-skew=false`
```

Comment:
@fejta I refer to existing CI test logs for the example commands.

Based on this [flag list](https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L107-L158), some of the flags may already be deprecated. For instance, the flag `disable-log-dump` is not in the list, but there is a `dump` flag [here](https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L117).

Reply:
I decided to remove this example due to the deprecated flag, and replaced it with a working example of mine. The newest commit already had the change.